packages/firestore/package.json

```diff
@@ -49,6 +46,7 @@
     "@firebase/app-types": "0.x"
   },
   "devDependencies": {
+    "@types/node": "8.10.59",
```

Comment:
This is in `devDependencies` in the root `package.json`, is there a reason to include it separately here?

Reply:
The paths in `externs.json` are relative to the package root, so I need this to be installed in packages/firestore/node_modules.

Reply:
Doesn't line 38 of `firestore/rollup.config.js` set them to be relative to repo root? The other files in `externs.json` seem to be relative to repo root.

Reply:
Yes! Updated.